src/patch_config.rs

```diff
@@ -553,6 +553,7 @@ pub struct PatchConfig
     pub results_string: Option<String>,
     pub artifact_hints: Option<HashMap<String,String>>, // e.g. "Strength":"This item can be found in Ruined Fountain"
     pub artifact_temple_layer_overrides: Option<HashMap<String,bool>>,
+    pub lore_hints: Option<HashMap<String,String>>,
```

Comment:
This variable name suggests the string replacements will be hints, when that is not necessarily the case.